packages/orchestrator-ui-components/src/components/WfoForms/formFields/BoolFieldStyling.ts

```diff
@@ -12,10 +12,6 @@ export const boolFieldStyling = css`
       z-index: unset;
     }
 
-    input[type="checkbox"] {
```

Comment:
This looks like it's put there intentionally so I am a bit worried we are going to introduce another problem. Any guesses to why it was put here in the first place?

Reply:
I share the worries, but according to the git history it seems introduced in the v1 implementation.